crypto/compat/posix_win.c

```diff
@@ -165,6 +165,7 @@ static void noop_handler(const wchar_t *expression,	const wchar_t *function,
 
 #define BEGIN_SUPPRESS_IPH \
 	int old_report_mode = _CrtSetReportMode(_CRT_ASSERT, 0); \
+	(void)old_report_mode; \
 	_invalid_parameter_handler old_handler = _set_thread_local_invalid_parameter_handler(noop_handler)
 #define END_SUPPRESS_IPH \
 	_CrtSetReportMode(_CRT_ASSERT, old_report_mode); \
```

Comment:
```suggestion
#define BEGIN_SUPPRESS_IPH \
	int old_report_mode = _CrtSetReportMode(_CRT_ASSERT, 0); \
	_invalid_parameter_handler old_handler = _set_thread_local_invalid_parameter_handler(noop_handler)
#define END_SUPPRESS_IPH \
	(void)old_report_mode; \
	_CrtSetReportMode(_CRT_ASSERT, old_report_mode); \
```
Wouldn't it be better to move the warning suppression into `END_SUPPRESS_IPH`? This way such warning would still correctly signal when missing an `END_SUPPRESS_IPH` by accident.

Reply:
Year I think that sounds good :)